### DISCUSSION TITLE
Enable Mouse Input

### DISCUSSION THREAD

**Discussion by ShadowXeldron** [Techokami/SonicWorldsNext#153] [Q&A]
This is probably a really stupid question, but how do I enable mouse input? I'm trying to make a menu with clickable objects but it seems to have been disabled.

**Comment by r6915ee** [comment]
This is more of a general Godot question, so most of it uses the regular GUI system Godot provides.

**Reply by ShadowXeldron** [reply]
I checked in a blank Godot project and the mouse as well as clickable buttons worked there. For whatever reason, they don't seem to work in Worlds Next.

**Reply by r6915ee** [reply]
Then perhaps there is a project setting that changes this functionality?
